brave/brave-core
# Add News custom RSS feed P3A questions, refactor News P3A

Resolves brave/brave-browser#21519

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Add two custom RSS feeds.
2. On the `brave://local-state` page, verify that the `Brave.Today.DirectFeedsTotal` metric has a value of 2.
3. Advance system time by two days.
4. Add five custom RSS feeds.
5. Verify that the `Brave.Today.DirectFeedsTotal` and `Brave.Today.WeeklyAddedDirectFeedsCount` metrics have a value of 6.
6. Advance system time by five days.
7. Verify that the `Brave.Today.DirectFeedsTotal` metric still has a value of 6 and `Brave.Today.DirectFeedsTotal` metric has a value of 5.
8. Advance system time by three days.
9. Verify that the `Brave.Today.DirectFeedsTotal` metric still has a value of 6 and `Brave.Today.DirectFeedsTotal` metric has a value of 0.
